front/src/pages/settings/SettingsObjectDetail.tsx

```diff
@@ -0,0 +1,15 @@
+import styled from '@emotion/styled';
+
+import { IconSettings } from '@/ui/icon';
+import { SubMenuTopBarContainer } from '@/ui/layout/components/SubMenuTopBarContainer';
+
+const StyledContainer = styled.div`
+  padding: ${({ theme }) => theme.spacing(8)};
+  width: 512px;
```

Comment:
I think we should export this as a constant as this will likely be used on all settings pages

Reply:
Done!